priv/perf/apps/load_test/lib/scenario/childchain_submit_transactions.ex

```diff
@@ -0,0 +1,87 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Scenario.ChildChainSubmitTransactions do
```

Comment:
hmmm slightly confused. Is this covered by the UTXO test done by @pgebal already?

Reply:
No, they're different scenarios.
This one attempts to submit txs to the childchain as fast as possible
The UTXO scenario attempts to gradually increase the total number of utxos in the childchain